client/src/app/site/tags/view-tag.ts

```diff
@@ -0,0 +1,56 @@
+import { Tag } from '../../shared/models/core/tag';
+import { TranslateService } from '@ngx-translate/core';
+import { BaseViewModel } from '../base/base-view-model';
+
+/**
+ * Category class for the View
+ *
+ * Stores a Category including all (implicit) references
+ * Provides "safe" access to variables and functions in {@link Category}
+ * @ignore
+ */
+export class ViewTag extends BaseViewModel {
```

Comment:
Duplicated.
ViewTag exists in `client/src/app/site/tags/models/view-tag.ts` aswell.